### DISCUSSION TITLE
GA4 and feedback buttons -- status?

### DISCUSSION THREAD

**Discussion by kimberleybrown** [google/docsy#1719] [Q&A]
Hello!

I recognize that there have been previous posts on the issue with GA4 and the feedback buttons and I was wondering if there has been progress in restructuring the feedback.html partial to integrate the gtag js code. As outlined in the original issue points, looking at the events in GA4 only shows the number of event clicks with the feedback button but does not distinguish which of the two buttons the user has selected. 

Curious what others have done and if could share if you've found a workaround that has been proven to be effective. 

Thanks!

**Comment by chalin** [comment]
Hi @kimberleybrown: No further progress to report, but Issue #1302 is scheduled for [23Q4](https://github.com/google/docsy/milestone/9) ✨

**Comment by jabhatfield** [comment]
How about a separate event for 'page_unhelpful'? Then it would be possible to create a free form report (Explore > Free form) with a row of 'page title', column of 'event name' and value of 'event count' (you may need to add these properties as dimensions and metrics first on the left hand side). This would give a view of helpful / unhelpful clicks for each page.

In terms of the current implementation it's possible to add an 'event value' column to the table in the 'pages and screens' report then filter the 'event count' column to 'page_helpful'. The sum of helpful (100) and unhelpful (0) can then be seen for each page (or change the first column to 'event name' to get a summary). This view is not especially intuitive though so an export to CSV ('Share this report') may be necessary in order to then calculate the percentage of positive responses.

**Comment by chalin** [comment]
Support landed in v[0.8.0](https://github.com/google/docsy/blob/main/CHANGELOG.md#080).

**Comment by chalin** [comment]
@jon-hatfield-tech-writing - thanks for your comment, I've moved it to a new thread so that we can carry on the discussion there. I'm closing this thread because it's been answered (the feature update has landed).
